File: src/_/06/get_glyph_legacy_opcode.ts
// SSoT: file:///Users/s0fractal/OMEGA/src/ontology/core/get_glyph_legacy_opcode.md
import { GLYPH_LEGACY_OPCODE_LUT } from "@g05";

export function get_glyph_legacy_opcode(id: number): number {
  return GLYPH_LEGACY_OPCODE_LUT[id & 63];
}

File: src/00/sigma_core/src/ontology_gen/06/get_glyph_legacy_opcode.rs
// SSoT: src/ontology/core/get_glyph_legacy_opcode.md
#![allow(unused_imports)]
use super::super::L05::*;

pub fn get_glyph_legacy_opcode(id: u8) -> u8 {
    GLYPH_LEGACY_OPCODE_LUT[(id & 63) as usize]
}
